identity/manager_test.go

```diff
@@ -3,16 +3,16 @@ package identity
 import (
 	"errors"
 	"testing"
-
 	"github.com/ethereum/go-ethereum/accounts"
 	"github.com/stretchr/testify/assert"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
```

Comment:
This unit was using mocked keystore, we should not need Ethereum library and real keystore in filesystem.

Reply:
If do, would be good good to separate such tests to `integration_test.go`

Reply:
We talked about this with @tadovas. It seemed logical to have some sort of checking whether the underlying ethereum keystore works. What if we update the eth keystore package and something breaks? We won't find out about it with mocks.

Reply:
Yes, agree on such integration test:
- 1 integration with happy path
- 10 units with rare edge cases